### DISCUSSION TITLE
In module federation, is there a way to know whether the current module is being used by the host application or by a remote application?

### DISCUSSION THREAD

**Discussion by AsaRory** [webpack/webpack#18795] [Q&A]
I have a file that is used by both the local project and by other applications through module federation. Is there any way to know in the code whether the current file is being used by other applications? For example, I want to judge in the code, `if (IS_USE_BY_MODULE_FEDERATION)`, and then perform certain operations.

**Comment by alexander-akait** [comment]
You should don't worry about it, why you need it?

**Comment by pladaria** [comment]
Until I get a better solution, in the host, before loading any remotes, I set:

`window.IS_HOST=true`

then in the remote module, I can check that variable
